package.json

```diff
@@ -75,7 +75,7 @@
     "js-base64": "^3.7.7",
     "js-md5": "^0.8.3",
     "lodash": "^4.17.21",
-    "marked": "^14.1.0",
+    "marked": "^15.0.7",
```

Comment:
Although the library supports global scope, I suggest using the instance approach: https://marked.js.org/using_advanced#instance.

Also, since we are making changes to [markdown.js](https://github.com/Adamant-im/adamant-im/blob/dev/src/lib/markdown.js#L1-L19), we can convert it to TypeScript.